fabric-fluids-v1/src/main/java/net/fabricmc/fabric/api/fluids/containers/FluidContainer.java

```diff
@@ -0,0 +1,57 @@
+package net.fabricmc.fabric.api.fluids.containers;
+
+import java.util.Iterator;
+
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.fluids.containers.volume.FluidVolume;
+import net.fabricmc.fabric.api.fluids.minecraft.FluidIds;
+
+/**
+ * An object that stores fluid, it is assumed that all fluid containers that only hold one fluid fluid Volumes
+ */
+public interface FluidContainer extends Iterable<FluidVolume> {
```

Comment:
Do we need more than one FluidVolume per FluidContainer ?

Reply:
wdym by that, do you mean "do we need multicontainers"? I'd say yes, there's a number of examples of that in forge mods, and it's nice to have

Reply:
just becuase its exists in forge doesnt mean we should have it

Reply:
so should I just remove fluid containers and use fluid volumes for everything